### PR TITLE
Add Benchmark Bash Script and README

### PR DESCRIPTION
**Pull request by Iajrdev**
This pull request includes the following updates:

- Addition of a benchmark bash script (`benchmark.sh`) to generate random numbers, insert them into server, and benchmark insert and find operations using `wrk`.
- 
- A README guide detailing the prerequisites, installation, execution steps, and script overview for the benchmark script.

### CODE REVIEW

File: BENCHMARK.md 
Comment (alipourhabibi) [inline BENCHMARK.md :1]: Please remove the trailing space at the end of file name so the github can detect this file as a markdown file

**Review by alipourhabibi — CHANGES_REQUESTED** [review]
In previous PR i forgot to add locking mechanism in v2 of the API
Can you please add the locking mechanism for them?

File: BENCHMARK.md 
Comment (Iajrdev) [reply BENCHMARK.md :1]: I removed the space form file name.

**Comment by Iajrdev** [comment]
> In previous PR i forgot to add locking mechanism in v2 of the API Can you please add the locking mechanism for them?

I have added locking mechanism to V2. Please review the code.

**Review by alipourhabibi — APPROVED** [review]

**Comment by alipourhabibi** [comment]
All looks good to me

**Pull request merged**